src/util/containers/ContainerManager.ts

```diff
@@ -0,0 +1,27 @@
+import type { ResourceIdentifier } from '../../ldp/representation/ResourceIdentifier';
+
+/**
+ * Handles everything related to container identifiers and how they relate to other identifiers.
+ */
+export interface ContainerManager {
```

Comment:
I wonder if it is actually a `ContainerStrategy`. It does not manage anything really.

Reply:
Or actually `IdentifierStrategy` even.

Reply:
Sure. Was also already thinking of changing the identifier generation when creating pods to also use this so makes sense.